index.html

```diff
@@ -10,19 +10,25 @@
     <header class="header">
         <div class="headerelement headerelement1">Welcome</div>
         <nav class="headerelement headerelement2"><img src="./Images/nav-menu1.png" height="25" width="25" alt="navimg"/></nav>
+        <nav class="header-desktop">
+            <div class="header-desktop1">Portfolio</div>
+            <div class="header-desktop2">About</div>
+            <div class="header-desktop3">Contact</div>
+        </nav>
     </header>
 
     <div class="headline">
         <h1><span class="p1">Hey there. I’m Juan</span><br><span class="p2">I’m a software<br>developer</span></h1>
+        <h2><span class="pd1">Hey there. I’m Juan</span><br><span class="pd2">I’m a software developer</span></h2>
         <p class="pheadline"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
             a project you need coded, don’t hestiate to contact me.</p>
     </div>
     <ul class="mediaimgs">
-        <li><i><img src="./Images/twitter.png" alt="twitter" height="20" width="22"/></i></li>
-        <li><i><img src="./Images/linkedin.png" alt="linkedin" height="20" width="22"/></i></li>
-        <li><i><img src="./Images/medium.png" alt="medium" height="20" width="22"/></i></li>
-        <li><i><img src="./Images/github.png" alt="github" height="20" width="22"/></i></li>
-        <li><i><img src="./Images/angellist.png" alt="angellist" height="20" width="22"/></i></li>
+        <li class="mediaimg1"><i><img src="./Images/twitter.png" alt="twitter" height="20" width="22"/></i></li>
+        <li class="mediaimg2"><i><img src="./Images/linkedin.png" alt="linkedin" height="20" width="22"/></i></li>
+        <li class="mediaimg3"><i><img src="./Images/medium.png" alt="medium" height="20" width="22"/></i></li>
+        <li class="mediaimg4"><i><img src="./Images/github.png" alt="github" height="20" width="22"/></i></li>
+        <li class="mediaimg5"><i><img src="./Images/angellist.png" alt="angellist" height="20" width="22"/></i></li>
```

Comment:
-It is a best practice to use CSS to style any HTML we write, hence I suggest you to write all your style inside CSS separated file

Reply:
Yes, you're right, I will change that immediately, Thank you @Tahiry Randriamiarintsoa.

All your recommendations were very good and help me to do it better, but the 2 changes you request are already done before the review.

In my desktop version, the changes you are suggesting... "In the header section, and in the screen size, The hamburger button shouldn't be displayed and the Item Navigation should be centered".

Here is a screenshot of my desktop version and it looks like the template, can't you see it right?

![2021-07-14 (1)](https://user-images.githubusercontent.com/82678907/125641285-52b32adb-eb89-4bc2-b42d-173f7211a97e.png)

And the second recommendation it's done too before the review, "The "My Recent Works"' section should be divided by 2 rows with 3 projects each (6 projects in total).. here is the screenshot..."

![2021-07-14 (2)](https://user-images.githubusercontent.com/82678907/125641410-52199f4d-af2e-41e1-91ed-dd4262907afd.png)

![2021-07-14 (3)](https://user-images.githubusercontent.com/82678907/125641411-6acef3f0-b62f-4161-9f89-55620e5ca513.png)

Maybe there is some confusion or the screen you see the project it's under 769px?.

 I have to ask for a second review, but as I said all the others recommendations were applied because you are totally right, thank you again.